www/flowjs.html

```diff
@@ -18,4 +18,6 @@
 	<script type="text/javascript" src="js/pixi.filters.js?10" defer></script>
 	<script type="text/javascript" src="js/jscommon.js?14" defer></script>
 	<script type="text/javascript" src="js/purify/purify.min.js?1" defer></script>
+	<script type="text/javascript" src="js/mark/mark.min.js" defer></script>
```

Comment:
It would be better to add them dynamically on first MMark usage (create a helper function in Haxe)